### PR TITLE
Add pdbfixer dependency to openff-toolkit-examples

### PR DESCRIPTION
**Pull request by Yoshanuikabundi**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

This PR adds the pdbfixer dependency to the openff-toolkit-examples package.

### CODE REVIEW

**Comment by Yoshanuikabundi** [comment]
@conda-forge-admin, please rerender

**Review by j-wags — APPROVED** [review]
LGTM, feel free to merge once tests pass!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -50,12 +52,17 @@ outputs:
 
     requirements:
       host:
-        - python >=3.6
+        - python >=3.8
       run:
-        - python >=3.6
+        - python >=3.8
+        - openmm >=7.6
+        - openff-interchange =0.2
```
Comment (mattwthompson) [inline recipe/meta.yaml:59]: Is there a plan to allow this to be installed alongside the newer version?

Comment (j-wags) [reply recipe/meta.yaml:59]: Nah, unintentional. I'll bump Interchange and Python for the 0.13 release. I can't recall a reason to bump openmm to >=8, could you remind me if there is one?

Comment (mattwthompson) [reply recipe/meta.yaml:59]: No compelling reason and some users might still be using things that require 7 (I think https://github.com/andysim/MPIDOpenMMPlugin is not updated, for example). We should probably communicate such a change before making it

**Pull request merged**